### PR TITLE
Burb

### PR DESCRIPTION
**Pull request by ALEXWARELLC**
Kill me.

I just require a URL to a hash file for the latest release of `exiled.tar.gz`.

### CODE REVIEW

File: Exiled.Installer/Program.cs [hunk 1]
```diff
@@ -174,6 +181,36 @@ private static string FormatRelease(Release r, bool includeAssets)
...
+        /// <param name="Hash">The downloaded file hash.</param>
+        /// <returns>Whether the validation succeeded.</returns>
+        
+        #pragma warning disable CS1998
+        private static async Task<bool> ValidateFileUsingWebsite(string Hash)
+        {
+            try
+            {
+                //TODO: Create a Installer Hash URL to get the latest app hash.
+                Task<HttpResponseMessage> response = new HttpClient().GetAsync("installer_hash_URL");
```
Comment (ALEXWARELLC) [inline Exiled.Installer/Program.cs:196]: Change the string to a URL to the hash file.

**Review by ALEXWARELLC — COMMENTED** [review]
Done a review,

**Review by louis1706 — CHANGES_REQUESTED** [review]
No.

**Comment by ALEXWARELLC** [comment]
> No.

OK.

**Pull request closed without merging**